docs/mcp_file_format.md

```diff
@@ -60,8 +60,25 @@ The `ServerRuntime` object specifies the transport protocol and its configuratio
 | Field | Type | Description | Required |
 |---|---|---|---|
 | `port` | integer | The port for the server to listen on. | Yes |
+| `basePath` | string | The base path for the MCP server. Defaults to `/mcp`. | No |
+| `auth` | `AuthConfig` | OAuth 2.0 configuration for protected resource. | No |
+| `tls` | `TLSConfig` | TLS configuration for the HTTP server. | No |
 
-### 4.2. StdioConfig Object
+### 4.2. TLSConfig Object
+
+| Field | Type | Description | Required |
+|---|---|---|---|
+| `certFile` | string | The absolute path to the server's public certificate file on the runtime host where the MCP server will execute. | Yes |
+| `keyFile` | string | The absolute path to the server's private key file on the runtime host where the MCP server will execute. | Yes |
+
+### 4.3. AuthConfig Object
+
+| Field | Type | Description | Required |
+|---|---|---|---|
+| `authorizationServers` | array of string | List of authorization server URLs for OAuth 2.0 token validation. | No |
+| `jwksUri` | string | JSON Web Key Set URI for token signature verification. | No |
```

Comment:
```suggestion
| `jwksUri` | string | JSON Web Key Set URI for token signature verification. If no value is given but `authorizationServers` is set, gen-mcp will try to find a JWKS endpoint using different fallback paths.| No |
```

Maybe we can add a note, that this is not required and what happens if this is skipped.